presto-testing/src/main/java/io/prestosql/testing/AbstractTestIntegrationSmokeTest.java

```diff
@@ -78,6 +80,12 @@ public void testAggregateSingleColumn()
         assertQuery("SELECT MAX(comment) FROM orders");
     }
 
+    // TODO: Remove this after we add support for creating table with columns having default values.
+    protected TestTable createTableWithDefaultColumns()
```

Comment:
leave this method `abstract` (any subclass should consciously provide impl or explicitly skip the test)

remove TODO above

move right after `testInsertForDefaultColumn` test